python/example_code/bedrock-agent/test/test_playlist_flow.py

```diff
@@ -0,0 +1,32 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.  
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import subprocess
+import sys
+
+files_under_test = [
+    "flows/playlist_flow.py"
+]
+
+@pytest.mark.integ
+@pytest.mark.parametrize("file", files_under_test)
+def test_playlist_flow(file):
+    # Simulate user input - each string represents one input() call
+    # If you're using the docs at https://docs.aws.amazon.com/bedrock/latest/userguide/flows-multi-turn-invocation.html,
+    # "Create a playlist\n 3\n pop, castles\n" should work with Antropic Haiku.
+    test_input = "pop\n3\n\y"  
```

Comment:
I'm getting a warning about the escape sequence here.
```suggestion
    test_input = "pop\n3\ny"  
```